pkg/agent/controller/networkpolicy/reconciler_test.go

```diff
@@ -161,12 +160,12 @@ func TestReconcilerReconcile(t *testing.T) {
 			},
 			[]*types.PolicyRule{
 				{
-					Direction:  networkingv1.PolicyTypeIngress,
+					Direction:  v1beta1.DirectionIn,
 					From:       ipsToOFAddresses(sets.NewString("1.1.1.1")),
 					ExceptFrom: nil,
 					To:         ofPortsToOFAddresses(sets.NewInt32(1)),
 					ExceptTo:   nil,
-					Service:    servicesToNetworkPolicyPort([]v1beta1.Service{serviceTCP80, serviceTCP}),
+					Service:    filterUnresolvablePort([]v1beta1.Service{serviceTCP80, serviceTCP}),
```

Comment:
no real need to call filterUnresolvablePort any more as the previous func was mainly used to do type conversion, but there's nothing to filter out obviously.